### PR TITLE
Fix multiarr js

### PR DESCRIPTION
**Pull request by ingRicardo**
Fix multiarr js

### CODE REVIEW

File: scriptTest.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+//Ricardo Alfredo Macias Olvera
+
+var arr =[1,23, [1, [ 2, [1] , 2]], [1]];
+console.log("Input -> "+arr);
+var myData = [];
+recu(arr);
+console.log("Output -> "+ myData);
+function recu(arr){
+
```
Comment (alejandrolechuga) [inline scriptTest.js:10]: Don't use global variables like myData, the function should take an input and give you an output

File: scriptTest.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+var arr =[1,23, [1, [ 2, [1] , 2]], [1]];
+console.log("Input -> "+arr);
+var myData = [];
+recu(arr);
+console.log("Output -> "+ myData);
+function recu(arr){
+   
+    for(let i =0; i<arr.length; i++){
+        if (typeof arr[i] == 'object'){
```
Comment (alejandrolechuga) [inline scriptTest.js:12]: to check if is an array you can do `Array.isArray(foo)`

**Review by alejandrolechuga — CHANGES_REQUESTED** [review]

File: scriptTest.js (hunk 1 above)
Comment (ingRicardo) [reply scriptTest.js:10]: done

File: scriptTest.js (hunk 2 above)
Comment (ingRicardo) [reply scriptTest.js:12]: done

**Comment by alejandrolechuga** [comment]
close this PR

**Pull request closed without merging**